src/index.js

```diff
@@ -40,6 +41,7 @@ if (perf && perf.mark && perf.measure) {
     return entries[entries.length - 1]
   }
   getEntries = () => perf.getEntriesByType('measure')
+  clearMeasures = () => perf.clearMeasures()
```

Comment:
I'd prefer for this to be `{ perf.clearMeasures() }` since there's no need to return anything, `performance.clearEntries()` returns undefined. Also I think `clearEntries` would be a clearer name since that's essentially waht it does, and it matches the `performance` API.